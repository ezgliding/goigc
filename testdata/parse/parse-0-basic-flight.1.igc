I033638FXA3940SIU4143ENL
J010812HDT
C150701213841160701000102500KTri
C5111359N00101899WEZ TAKEOFF
C5110179N00102644WEZ START
C5209092N00255227WEZ TP1
C5230147N00017612WEZ TP2
C5110179N00102644WEZ FINISH
C5111359N00101899WEZ LANDING
F160240040609123624
D20331
E160245ATS102312
B1602455107126N00149300WA002880042919509020
K16024800090
B1603105107212N00149174WV002930043519608024
LPLTLOG TEXT
GREJNGJERJKNJKRE31895478537H43982FJN9248F942389T433T
GJNJK2489IERGNV3089IVJE9GO398535J3894N358954983O0934
