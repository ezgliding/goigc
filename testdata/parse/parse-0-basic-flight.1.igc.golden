{
  "Manufacturer": "",
  "UniqueID": "",
  "AdditionalData": "",
  "Date": "0001-01-01T00:00:00Z",
  "FixAccuracy": 0,
  "Pilot": "",
  "Crew": "",
  "GliderType": "",
  "GliderID": "",
  "GPSDatum": "",
  "FirmwareVersion": "",
  "HardwareVersion": "",
  "FlightRecorder": "",
  "GPS": "",
  "PressureSensor": "",
  "CompetitionID": "",
  "CompetitionClass": "",
  "Timezone": 0,
  "Points": [
    {
      "Lat": 0.89219078789206,
      "Lng": -0.03179408120716337,
      "Time": "0000-01-01T16:02:45Z",
      "FixValidity": 65,
      "PressureAltitude": 288,
      "GNSSAltitude": 429,
      "IData": {
        "ENL": "020",
        "FXA": "195",
        "SIU": "09"
      },
      "NumSatellites": 6,
      "Description": ""
    },
    {
      "Lat": 0.8922158042780052,
      "Lng": -0.03175742929287149,
      "Time": "0000-01-01T16:03:10Z",
      "FixValidity": 86,
      "PressureAltitude": 293,
      "GNSSAltitude": 435,
      "IData": {
        "ENL": "024",
        "FXA": "196",
        "SIU": "08"
      },
      "NumSatellites": 6,
      "Description": ""
    }
  ],
  "K": [
    {
      "Time": "0000-01-01T16:02:48Z",
      "Fields": {
        "HDT": "00090"
      }
    }
  ],
  "Events": [
    {
      "Time": "0000-01-01T16:02:45Z",
      "Type": "ATS",
      "Data": "102312"
    }
  ],
  "Satellites": [
    {
      "Time": "0000-01-01T16:02:40Z",
      "Ids": [
        "04",
        "06",
        "09",
        "12",
        "36",
        "24"
      ]
    }
  ],
  "Logbook": [
    {
      "Type": "PLT",
      "Text": "LOG TEXT"
    }
  ],
  "Task": {
    "DeclarationDate": "2001-07-15T21:38:41Z",
    "Date": "2001-07-16T00:00:00Z",
    "Number": 1,
    "Takeoff": {
      "Lat": 0.8934221176793421,
      "Lng": -0.0180056892281995,
      "Time": "0001-01-01T00:00:00Z",
      "FixValidity": 0,
      "PressureAltitude": 0,
      "GNSSAltitude": 0,
      "IData": {},
      "NumSatellites": 0,
      "Description": "EZ TAKEOFF"
    },
    "Start": {
      "Lat": 0.8930788695931164,
      "Lng": -0.018222400943655463,
      "Time": "0001-01-01T00:00:00Z",
      "FixValidity": 0,
      "PressureAltitude": 0,
      "GNSSAltitude": 0,
      "IData": {},
      "NumSatellites": 0,
      "Description": "EZ START"
    },
    "Turnpoints": [
      {
        "Lat": 0.9102159666302401,
        "Lng": -0.050971468139868394,
        "Time": "0001-01-01T00:00:00Z",
        "FixValidity": 0,
        "PressureAltitude": 0,
        "GNSSAltitude": 0,
        "IData": {},
        "NumSatellites": 0,
        "Description": "EZ TP1"
      },
      {
        "Lat": 0.9163406178636969,
        "Lng": -0.0051231231310206885,
        "Time": "0001-01-01T00:00:00Z",
        "FixValidity": 0,
        "PressureAltitude": 0,
        "GNSSAltitude": 0,
        "IData": {},
        "NumSatellites": 0,
        "Description": "EZ TP2"
      }
    ],
    "Finish": {
      "Lat": 0.8930788695931164,
      "Lng": -0.018222400943655463,
      "Time": "0001-01-01T00:00:00Z",
      "FixValidity": 0,
      "PressureAltitude": 0,
      "GNSSAltitude": 0,
      "IData": {},
      "NumSatellites": 0,
      "Description": "EZ FINISH"
    },
    "Landing": {
      "Lat": 0.8934221176793421,
      "Lng": -0.0180056892281995,
      "Time": "0001-01-01T00:00:00Z",
      "FixValidity": 0,
      "PressureAltitude": 0,
      "GNSSAltitude": 0,
      "IData": {},
      "NumSatellites": 0,
      "Description": "EZ LANDING"
    },
    "Description": "500KTri"
  },
  "DGPSStationID": "0331",
  "Signature": "REJNGJERJKNJKRE31895478537H43982FJN9248F942389T433TJNJK2489IERGNV3089IVJE9GO398535J3894N358954983O0934"
}